cluster-autoscaler/cloudprovider/aws/aws_manager_test.go

```diff
@@ -103,6 +103,24 @@ func TestExtractAllocatableResourcesFromAsg(t *testing.T) {
 	assert.Equal(t, resource.NewQuantity(5, resource.DecimalSI).String(), labels["custom-resource"].String())
 }
 
+func TestExtractAllocatableResourcesFromTags(t *testing.T) {
```

Comment:
perhaps worth adding a test case that stresses the error branch for ParseQuantity?

Reply:
I have added a test case here that ensures an unparseable resource will be ignored in the output.